### DISCUSSION TITLE
Trouble with making Application

### DISCUSSION THREAD

**Discussion by Anastasiag2000** [UML-Linux-Club/moose-info#13] [Mac Install]
After updating conda and moose, I tried to remake the application following the guidlines from tutorial 1 and am still getting this error:

> ld: unsupported tapi file type '!tapi-tbd' in YAML file '/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib/libSystem.tbd' for architecture x86_64
> clang-9: error: linker command failed with exit code 1 (use -v to see invocation)
> make: *** [/Users/Anastasia/projects/moose/framework/../python/pyhit/hit.so] Error 1
> make: *** Waiting for unfinished jobs....

When I run tests, I get this error: 

> (moose) Anastasias-MacBook-Pro:diffusion Anastasia$ ./run_tests -j2
> failed to import pyhit - try running "make hit" in the $MOOSE_DIR/test directory.
> (moose) Anastasias-MacBook-Pro:diffusion Anastasia$ 
> 

Thoughts or insight are welcome.

**Comment by dealmeidavf** [comment]
Did the `moose` tests all passed? If so, could you go back and redo the tests and verify the installation is sound?
